doc/NPCs.md

```diff
@@ -827,7 +814,15 @@ Example | Description
 `"u_val": "stored_kcal_percentage"` | a value of 100 represents 55'000 kcal, which is considered healthy.
 `"u_val": "item_count"` | Number of a given item in the character's inventory. `"item"` must also be specified. Can be read but not written to.
 `"u_val": "exp"` | Total experience earned. Not supported for NPCs. Can be read but not written to.
-
+`"u_val": "stim"` | Current stim level.
+`"u_val": "pkill"` | Current painkiller level.
+`"u_val": "rad"` | Current radiation level.
+`"u_val": "focus"` | Current focus level.
+`"u_val": "fatigue"` | Current fatigue level.
+`"u_val": "stamina"` | Current stamina level.
+`"u_val": "sleep_deprivation"` | Current sleep deprivation level.
+`"u_val": "anger"` | Current anger level, only works for monsters.
```

Comment:
To avoid confusion with the NPC faction relationship anger statistic, perhaps "monster_anger" instead of just "anger"? Or could this be combined with accessing the NPC faction relationship?

OH. Can one currently check HP percentage of total? If not, that would be very nice, for all of monsters, NPCs, and the player. (For the latter two, possibly min(body %age, torso %age, head %age)?)

Reply:
So I had thought about a monster_ naming but the idea with talkers to some degree is to overlap when appropriate so I'm not sure.  The idea is a talker is generic so for morale it kinda makes sense that it should be the same thing for characters and monsters even if it doesn't behave the same. I'm on the fence.

Reply:
@Ramza13: One question is whether they're on the same scale, although I suppose one could compare vs their baseline values (0 for characters, variable by type for monsters).